src/test/java/seedu/address/logic/commands/AddFacebookContactCommandTest.java

```diff
@@ -0,0 +1,27 @@
+package seedu.address.logic.commands;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class AddFacebookContactCommandTest {
+
+    //private Model model = new ModelManager(getTypicalAddressBook(), new UserPrefs());
```

Comment:
Do remember to remove debugging or unused codes 👌✔

Reply:
ok thanks! leaving these code for future

Reply:
Okie dokie!